adr/configuration.py

```diff
@@ -117,12 +119,11 @@ def __init__(self, path=None):
         else:
             logger.warning(f"Configuration path {self.path} is not a file.")
 
-        self._config["sources"] = sorted(map(os.path.expanduser, set(self._config["sources"])))
+        self._config["sources"] = sorted(
+            map(os.path.expanduser, set(self._config["sources"]))
+        )
 
-        # Use the NullStore by default. This allows us to control whether
-        # caching is enabled or not at runtime.
-        self._config["cache"].setdefault("stores", {"null": {"driver": "null"}})
```

Comment:
I guess removing this would fix #103. I don't know if there are users of ADR relying on the current behavior (that is, still work even without a cache config)

Reply:
Oh, you actually moved it to the custom cache manager class, forget it.